src/mmrelay/meshtastic/node_refresh.py

```
@@ -0,0 +1,206 @@
+
+            user_snapshot: dict[str, Any] = {
+                "id": raw_user.get("id"),
+                PROTO_NODE_NAME_LONG: raw_user.get(PROTO_NODE_NAME_LONG),
+                PROTO_NODE_NAME_SHORT: raw_user.get(PROTO_NODE_NAME_SHORT),
+            }
+            nodes_snapshot[node_key] = {"user": user_snapshot}
+
+        return nodes_snapshot, False
+
+
+async def refresh_node_name_tables(
```

Ruff (0.15.9)

[warning] Too many branches (16 > 12)

(PLR0912)